commands/curl_delete.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const curlDeleteCommandName = "delete"
+
+var curlDeleteExample = `
+# Delete a document from an index. 
+odfe-cli curl delete --path         "my-index/_doc/1" \
+                     --query-params "routing=odfe-node1"
+`
+
+var curlDeleteCmd = &cobra.Command{
```

Comment:
Might be nice to add safe mode to delete command: like "Are you sure you want to ...". Maybe a feature for the future.

Reply:
Sure. Let me look into it.